custom/panel_templates/Default/template.php

```diff
@@ -155,11 +155,11 @@ public function onPageLoad() {
                             AssetTree::DATATABLES
                         ]);
                         $this->addCSSFiles([
-                            (defined('CONFIG_PATH') ? CONFIG_PATH : '') . '/custom/panel_templates/Default/assets/css/fomantic.toast.min.css' => [],
```

Comment:
Now that this is being used in the panel template and frontend template, we should probably use yarn for it